packages/api-client/src/replays.ts

```diff
@@ -29,7 +29,7 @@ export const postKinesisReplays = async (params: {
       headers: {
```

Comment:
Do we still expect the payload above to be MigrationCountsPayload? I think we can create a new type and add `kinesisStream`